addons/mass_mailing/data/mailing_data_templates.xml

```diff
@@ -91,35 +91,37 @@
     <template id="ab_testing_description" name="Mass Mailing: A/B Test Description">
```

Comment:
Side-note: it's better to always refer to the task like this: `Task-3008627`.

If your number matches a PR or a commit hash, github may try to link it.

Side-side-note:

> Since then a 100% limit for the recipients of an A/B campaign.

I think this sentence is missing something ^^

Reply:
:+1: took note of that the last time you mentioned it but I guess this task is older